presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSpatialPartitioningInternalAggregation.java

```diff
@@ -93,6 +95,23 @@ public void test(int partitionCount)
         assertEquals(groupValue, expectedValue);
     }
 
+    @Test
+    public void testEmptyPartitionException()
+    {
+        InternalAggregationFunction function = getFunction();
+
+        int partitionCount = 10;
+        List<OGCGeometry> geometries = ImmutableList.of();
+        Block geometryBlock = makeGeometryBlock(geometries);
+        Block partitionCountBlock = BlockAssertions.createRLEBlock(partitionCount, geometries.size());
```

Comment:
- `partitionCount` and `geometries` variables are not needed
- no need to use `makeGeometryBlock` method

```
        Block geometryBlock = GEOMETRY.createBlockBuilder(null, 0).build();
        Block partitionCountBlock = BlockAssertions.createRLEBlock(10, 0);
```